tests/SpartaHack_API_2019.postman_collection.json

```diff
@@ -0,0 +1,37 @@
+{
+	"info": {
+		"_postman_id": "53580b40-b7fe-4716-ad25-690205537b68",
+		"name": "SpartaHack_API_2019",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "faqs",
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					},
+					{
+						"key": "X-WWW-USER-TOKEN",
+						"value": "mrXeSfPqssiicUuRF3_2"
```

Comment:
Where did you get this token from? Is it a user token that probably shouldn't be shown?

Reply:
It's fine. it's an artifact from the old API. I just copied the headers from the old postman collection. I am not even using it. I need to figure out a way to handle these or live with the fact that one user token on the dev server is going to be public. Which isn't that much of a risk because first it's not from prod second I am going to apply rate limiter which would help with DDOS attacks.